docker/services/ensdb.yml

```
@@ -1,13 +1,10 @@
 services:
   ensdb:
-    container_name: ensdb
     image: postgres:17
-    ports:
-      - "5432:5432"
     volumes:
       - ensdb_data:/var/lib/postgresql/data
     healthcheck:
-      test: ["CMD-SHELL", "pg_isready -U $${POSTGRES_USER} -d $${POSTGRES_DB}"]
+      test: [ "CMD-SHELL", "pg_isready -U $${POSTGRES_USER} -d $${POSTGRES_DB}" ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)